bazel/envoy_test.bzl

```diff
@@ -138,6 +134,32 @@ def envoy_cc_fuzz_test(
         tags = ["fuzz_target"] + tags,
     )
 
+    fuzzing_corpus(
+        name = name + "_corpus_dir",
+        srcs = [corpus_name],
+        testonly = True,
+    )
+
+    # Target for continuous fuzzing test or regression gUnit test
+    fuzzing_launcher(
+        name = name + "_run",
+        target = name + "_binary",
+        corpus = name + "_corpus_dir" if corpus_name else None,
+        is_regression = select({
+            "@envoy//bazel:libfuzzer": False,
+            "//conditions:default": True,
+        }),
```

Comment:
Do we need to override this here? Maybe we can even remove the `is_regression` attribute altogether from the launcher rule and only pass it directly to the launcher binary as a command line argument (e.g., from the shell script).

Reply:
The original `is_regression` is set to `True`, thus this comment is not fully solved until this [PR](https://github.com/googleinterns/bazel-rules-fuzzing/pull/65) is merged and Envoy is updated with the new version of Bazel-rules-fuzzing.